hack/pre-release/pre-release-tpa.sh

```
@@ -0,0 +1,267 @@
+
+parse_args() {
+    init_github_token
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+        -t|--github-token)
+            if [[ -z "${2:-}" ]]; then
+                echo "[ERROR] GitHub token needs to be specified after '--github-token'." >&2
+                usage
+                exit 1
+            fi
+            GITHUB_TOKEN="$2"
```

Shellcheck (0.11.0)

[warning] GITHUB_TOKEN appears unused. Verify use (or export if used externally).

(SC2034)